_posts/2022-02-23-roadmap-proposal.md

```diff
@@ -0,0 +1,76 @@
+
+
+
+### Proposed schedule
+
+Taking the above into consideration, here’s how I think the majors and minors would lay out on the calendar.  If you’re planning on submitting a code change, note the corresponding feature freeze dates (the date when no more changes can be added so the release can be tested). 
+
+|Version	|Feature Freeze	|Release Date	|
+|-------------	|-------------	|-------------	|
+|1.3.0	|March 9th	|March 15th	|
+|2.0.0 Preview Release	|March 21st	|March 31st	|
+|1.4.0	|April 20th	|April 26th	|
+|2.0.0 GA	|May 2nd	|May 12th	|
+|2.1.0	|June 23rd	|June 30th	|
+|2.2.0	|August 4th	|August 11th	|
+|3.0.0 Preview Release	|September 5th	|September 14th	|
+|2.4.0	|September 22nd	|September 29th	|
+|2.5.0	|December 1st	|December 8th	|
+|3.0.0 GA	|January 10th, 2023	|January 19th 2023	|
+{: .opensearch-supported-versions}
+
+You’ll notice I’ve included a "Preview Release" for our major releases in 2022 to help folks get an early look. What the exact release process will look like and how feedback for the previews can be identified is yet-to-be determined, but as the first major releases I want to make sure everyone has a chance to try them out early.
+
+Also, I’d like to increase how often OpenSearch is able to release minor version from 6 weeks to 4 by the end of the year and reduce the length of feature freeze required, but that will depend on our ability to add automation. For now I’ve planned with 6 week intervals, but when the team is ready I’ll reset the dates.
+
+### Themes for the Major releases
+
+#### 2.0.0
+
+As mentioned above, the primary driver for the team to have an earlier 2.0.0 release is so that OpenSearch can get support for Lucene 9.0.0 in earlier. This change will likely enable some use of new Lucene features within the plugins, like KNN and Vector Field types. This release also allows for other breaking changes to be ready earlier, like inclusive naming.  
+
+#### 3.0.0
+
+Some 2022 themes for the core of OpenSearch:  Security, Efficiency, Durability, Extensibility, and Engagement. This is the release where those ideals are translated into changes you can use: Things like [segment replication](https://github.com/opensearch-project/OpenSearch/issues/1694), [inbuilt security](https://github.com/opensearch-project/OpenSearch/issues/1029) and [sandboxing](https://github.com/opensearch-project/OpenSearch/issues/1422).  I’m excited to work with you to make those improvements, and to see your thoughts on how to make things even better. 
+
+### One final note....
+
+Like all schedules, it’s possible that it will need to change based on real world conditions *cough* log4j *cough*. As the year goes along, if better way to do things are found, or someone has a brilliant idea for a feature, those changes will be made.  If nothing else, in January of 2023 everyone can look back at this schedule with a wry smile at the bold confidence of February 2022.
+
+### Feedback?
+
```

Comment:
I don't think we allow comments on blog posts, do we?  Maybe we should open up a forum post so folks can ask questions?  It would also give us a known location to communicate any date changes.  Otherwise we should add a link in the OpenSource-project/OpenSearch issues to make it easy to cut an issue.

Reply:
Sounds good. I'll make that change.